config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits.yaml

```diff
@@ -13,3 +13,87 @@ presubmits:
         - runner.sh
         args:
         - ./scripts/ci-test.sh
+  
+  - name: pull-cluster-api-provider-vsphere-e2e
+    labels:
+      preset-dind-enabled: "true"
+      preset-service-account: "true"
+    always_run: true
+    max_concurrency: 1
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181120-dea0825e3-master
+        args:
+        - "--job=$(JOB_NAME)"
+        - "--root=/go/src"
+        - "--repo=sigs.k8s.io/cluster-api-provider-vsphere=$(PULL_REFS)"
+        - "--repo=figo/cluster-api-provider-vsphere-ci=master"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--scenario=execute"
```

Comment:
if you are using the execute scenario, please use the pod-utils instead, the flags here go to a legacy script (jenkins/bootstrap.py) that we intend to remove in the future
https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md#writing-a-prowjob-that-uses-pod-utilities

Reply:
@BenTheElder  thanks, looking at it.